backend/requirements.txt

```
@@ -17,6 +17,7 @@ pytest==6.2.4
 PyYAML==5.4.1
 tqdm==4.62.1
 langdetect==1.0.8
+isodate
```

The version used should be specified, to avoid potential breakage when a new version is available:
```suggestion
isodate==0.6.0
```